x/epochs/abci.go

```diff
@@ -0,0 +1,58 @@
+package epochs
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/telemetry"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tharsis/evmos/x/epochs/keeper"
+	"github.com/tharsis/evmos/x/epochs/types"
+)
+
+// BeginBlocker of epochs module
+func BeginBlocker(ctx sdk.Context, k keeper.Keeper) {
+	defer telemetry.ModuleMeasureSince(types.ModuleName, time.Now(), telemetry.MetricKeyBeginBlocker)
+	k.IterateEpochInfo(ctx, func(index int64, epochInfo types.EpochInfo) (stop bool) {
```

Comment:
@ramacarlucho Don't we just have one epoch? Why do we need to iterate?

Reply:
Its designed to allow more than one epoch